spec/presenters/finder_presenter_spec.rb

```diff
@@ -60,6 +71,21 @@
     it "exposes the government_content_section" do
       government_presenter.government_content_section.should == "policies"
     end
+
+    it "has metadata" do
+      government_presenter.any_page_metadata?.should be_true
```

Comment:
There are a couple of RSpec matchers which can make testing booleans a bit smoother: https://github.com/rspec/rspec-expectations#predicate-matchers

To use it in this case would probably require renaming the method to `has_page_metadata?`, which would then allow this test to be:

```ruby
it "has metadata" do
  government_presenter.should have_page_metadata
end
```

For the negative case swap the `should` for `should_not`.

Whether you think that's an improvement worth having I'll leave up to you to decide.

One other thing I think it's worth noting: the `should` syntax is deprecated in RSpec 3 in favour of the `expect` syntax, in order to avoid monkey patching `Object`. This code base mixes styles already, so again I'll leave it to you to decide what to do with that information here.

```
it "has metadata" do
  expect(government_presenter).to have_page_metadata
end
```